charts/kapacitor/TICKscripts/test_scripts/num_containers_stopped_tester.tick

```diff
@@ -0,0 +1,16 @@
+//dbrp "telegraf"."autogen"
```

Comment:
Does the test fail/not work when this is uncommented? [This PR](https://github.com/gpestana/kapacitor-unit/pull/30) supposedly fixed the problem here.

Reply:
Yeah, I just haven't updated kapacitor-unit on my VM. I figure it's better to comment it out since the engineers are probubly running older versions.

Reply:
Okay. Try to make a note about this in the documentation you write.